pkg/features/list.go

```diff
@@ -51,4 +51,7 @@ var (
 
 	// ResyncDisabled disables the resync behavior of the kubernetes listeners in sensor
 	ResyncDisabled = registerFeature("Disable the re-sync", "ROX_RESYNC_DISABLED", false)
+
+	// SyslogextraFields enables user to add additional key value pairs in syslog alert notification in cef format
+	SyslogextraFields = registerFeature("Enable extra fields for syslog integration", "ROX_SYSLOG_FIELDS", false)
```

Comment:
Nit: SyslogExtraFields  and ROX_SYSLOG_EXTRA_FIELDS

Reply:
done